### DISCUSSION TITLE
How to alter the sorting drop down Products Display List?

### DISCUSSION THREAD

**Discussion by gnossosp** [openstore-ecommerce/OpenStore#197] [Q&A] [answered]
I would like to have control over sorting the initial (Default) sorting on the product display list when page loads. Where/How can I manipulate this feature? thanks

**Comment by DNNMonster** [comment, marked as answer]
There is a known issue with default sorting related to first rendering.  I will try to make sure this gets taken care of.

**Reply by gnossosp** [reply]
Thanks!
